__tests__/fakes.ts

```diff
@@ -50,3 +48,19 @@ export class FakeDependencies implements Dependencies {
         this.failCallHistory.push({ failMessage })
     }
 }
+
+export class FakeCommandExecutor implements CommandExecutor {
+    isSalesforceCliInstalledCallCount = 0
+    isSalesforceCliInstalledReturnValue = true
+    async isSalesforceCliInstalled(): Promise<boolean> {
+        this.isSalesforceCliInstalledCallCount++
+        return this.isSalesforceCliInstalledReturnValue
+    }
+
+    runCodeAnalyzerReturnValue = 0
+    runCodeAnalyzerCallHistory: { runCmd: string; runArgs: string; internalOutfile: string }[] = []
+    async runCodeAnalyzer(runCmd: string, runArgs: string, internalOutfile: string): Promise<number> {
```

Comment:
If the linter runs against this file, it'll complain about having an `async` method without a corresponding `await`.

Reply:
Actually, our linter is different than the default one used for typescript actions based off of this template: https://github.com/actions/typescript-action 

Many linters run for this repo:
* And the main one with the ./github/workflows/ci.yml workflow (which does `npm run lint`)
* A number of other ones with the ./github/workflows/linter.yml workflow (which uses super-linter/super-linter/slim@v5)

I'd prefer to use the templated standards here.